user-guide/Reference/faq/General_configuration.md

```diff
@@ -58,19 +58,28 @@ More information about SNTP can be found in RFC 4330:
 
 The time server will provide the correct time settings for all DMAs.
 
-To turn any computer into a time server, whether it serves as a DMA or not, do the following.
+To turn any computer using Windows Server 2008 R2, Windows 7 or later into a time server, whether it serves as a DMA or not, do the following:
 
 1. Log on locally to the machine that will act as time server, or establish a remote desktop session with it.
 
-1. Open Registry Editor (*Start \> Run... \> regedit*).
+1. Stop the "Windows Time" service via Start \> Control Panel \> System and Security \> Administrative Tools \> Services.
 
-1. Go to the following key:
+1. Open the registry editor via Start \> Run... \> regedit.
```

Comment:
Again, please use italics for direct references to UI.

Reply:
done